### ISSUE TITLE
Editing WG

### ISSUE THREAD

**Issue by wseltzer** [w3c/strategy#255]
https://github.com/w3c/webappswg/issues/42

Advance notice sent: https://lists.w3.org/Archives/Public/public-new-work/2021Feb/0006.html

Draft charter: https://w3c.github.io/editing/charter-drafts/editing-2021.html

**Comment by wseltzer** [comment]
Advance notice sent: https://lists.w3.org/Archives/Public/public-new-work/2021Feb/0006.html

**Comment by siusin** [comment]
New charter proposal, reviewers please take note.

# Charter Review

[Editing Working Group Charter:](https://w3c.github.io/editing/charter-drafts/editing-2021.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* New
 - [x] [New WG](https://lists.w3.org/Archives/Public/public-new-work/2021Feb/0006.html) 
 - [ ] New IG

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security.  Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).


Communities suggested for outreach: 
Nope.

Known or potential areas of concern: 
Nope.

Where would charter proponents like to see issues raised?
Please open an issue in the [w3c/editing repo](https://github.com/w3c/editing/).

Anything else we should think about as we review? 
Nope.

Thank you!

**Comment by samuelweiler** [comment]
I see lots of unfixed "todo" items in the charter, yet this draft misses some of the changes that are in the [current template](https://github.com/w3c/charter-drafts/blob/gh-pages/charter-template.html) including the line re: security and privacy sections.  I'm not sure what else might be missing.  Your choice of whether to backport the change(s) or restart from the current template. 

There is also some formatting cleanup to be done in the deliverables list.

Full review yet to come.

**Comment by michael-n-cooper** [comment]
APA WG wants a liaison statement, as editing has many accessibility considerations yet to be addressed. Proposed description: "Explore accessibility use cases and gap analysis and ensure new specifications provide needed accessibility features."

**Comment by michael-n-cooper** [comment]
APA review complete, over to @brewerj to complete accessibility horizontal review.

**Comment by siusin** [comment]
> I see lots of unfixed "todo" items in the charter, yet this draft misses some of the changes that are in the [current template](https://github.com/w3c/charter-drafts/blob/gh-pages/charter-template.html) including the line re: security and privacy sections (and a different odd line mentioning privacy). I'm not sure what else might be missing. Your choice of whether to backport the change(s) or restart from the current template.
> 
> There is also some formatting cleanup to be done in the deliverables list.
> 
> Please tag me here when the security and privacy line(s) are fixed.

Thanks Sam. It should be fixed by [PR#284](https://github.com/w3c/editing/pull/284).

**Comment by samuelweiler** [comment]
> > I see lots of unfixed "todo" items in the charter, yet this draft misses some of the changes that are in the [current template](https://github.com/w3c/charter-drafts/blob/gh-pages/charter-template.html) including the line re: security and privacy sections (and a different odd line mentioning privacy). I'm not sure what else might be missing. Your choice of whether to backport the change(s) or restart from the current template.
> > There is also some formatting cleanup to be done in the deliverables list.
> > Please tag me here when the security and privacy line(s) are fixed.
> 
> Thanks Sam. The todo items should be fixed by [PR#284](https://github.com/w3c/editing/pull/284).

todo items look pretty good; the security and privacy text still does not match the template.

**Comment by travisleithead** [comment]
I'd like to tighten the scope section so that it is focused on web scenarios, not editing scenarios of all kinds and flavors. Instead of:

> The scope of Editing WG covers all aspects of text editing, which may include:

I'd like to propose:

> The scope of Editing WG covers all aspects of text editing **on the web**, which may include:

**Comment by himorin** [comment]
From i18n WG, we also would like to request to be listed in 4.1 coordination section, that there could be many considerations on internationalization point of view to be addressed in editing especially for edit/input, keyboard, etc.

**Comment by siusin** [comment]
Thanks for the review comments. We've updated the draft charter in [#286](https://github.com/w3c/editing/pull/286) and [#287](https://github.com/w3c/editing/pull/287).

Latest version:
  https://w3c.github.io/editing/charter-drafts/editing-2021.html

**Comment by brewerj** [comment]
Hi @siusin Thank you for adding the liaison sentence that Michael Cooper requested. 

Additionally, given that editing is a form of authoring, a reference to the [Authoring Tool Accessibility Guidelines 2.0](https://www.w3.org/TR/ATAG20/) would be relevant. I would not expect this to be a full dependency nor the Editing WG deliverables to necessarily go into depth on accessibility aspects of authoring, but think that the WG should be aware of ATAG 2.0, and consider sections to point and/or incorporate where appropriate. 

Thanks, and please let me know if you have questions.

**Comment by siusin** [comment]
Thanks @brewerj ! 

Is this a request to the specs or to the charter? It seems the ATAG WG has already been closed.

**Comment by brewerj** [comment]
@siusin It is a request to reference the ATAG 2.0 Recommendation that I provided a link to in my comment above. 

The scope of the former ATAG WG has in part been carried into the Accessibility Guidelines Working Group, but the relevant bit for your charter is what I named and linked to above. Please let me if you have further questions; thanks.

**Comment by plehegar** [comment]
We could add, in section 4.1:
[[
Accessibility Guidelines Working Group
        The Accessibility Guidelines Working Group (AG WG) maintains the Authoring Tool Accessibility Guidelines (ATAG) 2.0, which provides guidelines for how authoring tools and environments can support accessibility for authors with disabilities, as well as promoting production of accessible content by all authors.
]]

It doesn't a requirement but a reminder of the work.

**Comment by brewerj** [comment]
@plehegar thanks for the discussion on how to add in a relevant reference to ATAG here. I think that a dependency with AG WG in this case makes sense, and a discussion with the Editing WG at some point in the first few months of their work @michael-n-cooper  I can help with that discussion, and we can potentially bring in one or two other people on that discussion as well. @siusin please let us know if you have questions. Thanks!

**Comment by siusin** [comment]
@brewerj AG WG is now [listed in section 4.1](https://w3c.github.io/editing/charter-drafts/editing-2021.html#coordination) and the co-chairs are aware that the AG WG would like to have a joint meeting in the next few months if we are able to create the group. Thanks for the comments :)

**Comment by samuelweiler** [comment]
I think this was closed in error.

**Comment by samuelweiler** [comment]
Thank you for updating the security and privacy text to match the template.

I'm used to seeing the list of deliverables in charters include a brief summary of each document, e.g. stolen from the doc's abstract.  I particularly note the lack of that in the "Other Deliverables" section, where we don't have links to the documents.  

I'm also aware that our charters are too wordy to begin with.  Up to you whether to fix that for the existing deliverables, but I do think more words are needed for the "other deliverables", unless you have links available for not-yet-adopted proposed docs.

Security and privacy reviews complete.

**Comment by dontcallmedom** [comment]
Group created https://lists.w3.org/Archives/Public/public-new-work/2021Jun/0007.html
